pkg/cmd/install.go

```diff
@@ -799,10 +800,22 @@ func installCommand() *cli.Command {
 					return err // we want the user to see the error message without a prefix
 				}
 			}
+
+			if err := maybePromptForAppUpdate(c, prompts.New(), license); err != nil {
+				if errors.Is(err, ErrNothingElseToAdd) {
+					metrics.ReportApplyFinished(c, err)
+					return err
+				}
+				// If we get an error other than ErrNothingElseToAdd, we warn and continue as
+				// this check is not critical.
+				logrus.Infof("WARNING: Failed to check for newer app versions: %v", err)
```

Comment:
My comment might not have been clear. I think `logrus.Infof` still prints to the terminal. I'm saying that if we fail to perform this check, that should be silent to the user, because they can't really do anything about that, and it won't affect their installation. I think printing a warning will be construed as "something went wrong" and cause people to reach out to us.

Reply:
Sorry it was clear. I didnt realize info printed to the terminal. I will change to debug

Reply:
updated